web/ui/react-app/src/pages/targets/ScrapePoolPanel.tsx

```diff
@@ -14,6 +14,19 @@ interface PanelProps {
   targetGroup: ScrapePool;
 }
 
+export const ToggleButtonProps = {
```

Comment:
If you want to share common styles and behavior its better to create a component instead.